aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -71,6 +71,21 @@ Example `HostBuilder` configuration using `ConfigureHostConfiguration`:
 
 Extension methods are called on the `IHostBuilder` implementation to configure the content root and the environment.
 
+#### Application Key (Name)
+
+The [IHostingEnvironment.ApplicationName](/dotnet/api/microsoft.extensions.hosting.ihostingenvironment.applicationname) property is set from host configuration during host construction. To set the value explicitly, use the [HostDefaults.ApplicationKey](/dotnet/api/microsoft.extensions.hosting.hostdefaults.applicationkey):
+
+**Key**: applicationName  
+**Type**: *string*  
+**Default**: The name of the assembly containing the app's entry point.  
+**Set using**: `UseSetting`  
+**Environment variable**: `<PREFIX_>APPLICATIONKEY` (`<PREFIX_>` is [optional and user-defined](#configuration-builder))
+
+```csharp
+WebHost.CreateDefaultBuilder(args)
+    .UseSetting(WebHostDefaults.ApplicationKey, "CustomApplicationName")
+```
+
 #### Content Root
```

Comment:
Convert title to sentence case while you're in here.

Reply:
These all mirror their key names, so I was treating all of them as proper nouns.